service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsAboutToSubmitControllerTest.java

```diff
@@ -95,11 +102,10 @@ private void assertC2BundleDocument(C2DocumentBundle documentBundle, String desc
         assertThat(documentBundle.getDescription()).isEqualTo(description);
 
         List<SupportingEvidenceBundle> supportingBundle = unwrapElements(documentBundle.getSupportingEvidenceBundle());
-        if (!supportingBundle.isEmpty()) {
-            SupportingEvidenceBundle supportingEvidenceBundle = supportingBundle.get(0);
-            assertThat(supportingEvidenceBundle.getName()).isEqualTo("Supporting document");
-            assertThat(supportingEvidenceBundle.getNotes()).isEqualTo("Document notes");
-        }
+
+        SupportingEvidenceBundle supportingEvidenceBundle = supportingBundle.get(0);
```

Comment:
This is not ideal, as in case of the system not properly replying it will file with indexoutofbounds exception.

I would expect explicit assertion ensuring there is one element before reaching for it. But it's good enough :)

Reply:
added check for not empty